pkg/qemu/imgutil/imgutil.go

```diff
@@ -17,7 +17,19 @@ type Info struct {
 
 func QCOWToRaw(source string, dest string) error {
 	var stdout, stderr bytes.Buffer
-	cmd := exec.Command("qemu-img", "convert", source, dest)
+	cmd := exec.Command("qemu-img", "convert", "-f", "qcow2", "-O", "raw", source, dest)
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("failed to run %v: stdout=%q, stderr=%q: %w",
+			cmd.Args, stdout.String(), stderr.String(), err)
+	}
+	return nil
+}
```

Comment:
Why do we still need this function?

Reply:
Now that you mention it, we probably don't. It's only useful if we want to explicitly specify that the input is qcow2, but qemu-img should be able to detect that on its own. Updated in latest rev.